coremltools/converters/mil/mil/ops/defs/control_flow.py

```diff
@@ -460,6 +542,30 @@ def type_inference(self):
 
 @register_op(doc_str="")
 class list_read(Operation):
+    """
+    Read the value at location ``index`` of ``ls``.
+    
+    Parameters
+    ----------
+    ls: List[*] (Required)
+    
+    index: <i32>  (Required)
+        * Size of the list.
+    
+    value: <*,T> (Optional)
```

Comment:
list_read doesn't take `value` as an argument.

Reply:
Thanks! Fixed in next commit.